jwt/api_jwt.py

```diff
@@ -87,8 +87,13 @@ def _validate_claims(self, payload, options, audience=None, issuer=None,
         if isinstance(leeway, timedelta):
             leeway = timedelta_total_seconds(leeway)
 
-        if not isinstance(audience, (string_types, type(None))):
-            raise TypeError('audience must be a string or None')
+        if not isinstance(audience, (string_types, type(None), list)):
+            if not isinstance(audience, list):
+                raise TypeError('audience must be a string or None')
```

Comment:
What do you think about moving this all into `self._validate_aud()`?

That would probably simplify the implementation a little bit.